src/kani-compiler/rustc_codegen_kani/src/context/metadata.rs

```diff
@@ -17,6 +17,8 @@ pub struct HarnessMetadata {
     pub original_file: String,
     /// The line in that file where the proof harness begins
     pub original_line: String,
+    /// Optional data to store unwind value
+    pub unwind_value: Option<u128>,
```

Comment:
I thought we agreed having a specific struct for unwind data where this would be an optional value. Also, `Option<u32>` should be enough.

Reply:
A struct didn't seem necessary for now as we're just targeting a default value for the entire harness. The struct and related changes will be added with the changes to map each loop to it's own harness.

Reply:
Maybe create a type alias here that we can extend later.

```rust
type UnwindMetadata = usize;
```

Reply:
do we want`usize` or `u32/64`. Doesnt `usize` tie the limit of the unwind value to the target architecture?

Reply:
I suggested `usize` because that's usually used to restrict structure sizes, but in  practice, I highly doubt someone is going to use a large unwind parameter. So `u32` is fine too.

Reply:
IMO stay an integer. Any future unwinding information (e.g. unwindset) can be provided in separate fields. We're never going to change `unwind_value` because it's just the thing we pass to cbmc as `--unwind` and that's it.

Reply:
I don't know why any future support for those wouldn't just add an `unwind_max_value` as well. There doesn't seem to be a value to nesting it. We control the producer (kani-compiler) and consumer (cargo-kani) of this structure, so if it's looks like a mistake later, we can just fix it. So simplest choice seems best to me.

But I don't feel strongly about it.